apps/test/unit/lib/tools/jsdebugger/JsDebuggerTest.js

```diff
@@ -124,22 +126,26 @@ describe('The JSDebugger component', () => {
       });
 
       it('will expand the debugger by setting the height in the css', () => {
-        expect(debugAreaEl().style.height).to.equal('120px');
+        expect(debugAreaEl().props().style.height).to.equal(120);
```

Comment:
> by setting the height in the css

I'm not sure this is true anymore, how the test is now written.

Reply:
will fix